versioning-strategies/accept-version.js

```diff
@@ -0,0 +1,9 @@
+const SemVerStore = require('semver-store')
+
+module.exports = {
+  name: 'ACCEPT_VERSION',
+  storage: SemVerStore,
+  deriveVersion: (req, ctx) => {
```

Comment:
Can you use a normal function here? I don't think it's needed, but I'm being paranoic about perf ;).

Reply:
done, tnx